samples/nodejs-compat-streams-split2/split2.js

```diff
@@ -16,128 +16,133 @@ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
 IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 */
 
-'use strict'
+'use strict';
```

Comment:
This file should likely be left as is and ignored. It was copied from the module and isn't our code.

Reply:
Added to .prettierignore